OctopusDSC/Tests/cOctopusServer.Tests.ps1

```diff
@@ -211,17 +211,26 @@ try
                         { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3"} | Should not throw
                     }
                     It "Should not throw if 'OctopusAdminCredential' not supplied" {
-                        { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3" -SqlDbConnectionString "blah4" -OctopusAdminCredential $null} | Should not throw
+                        { Test-ParameterSet -Ensure 'Present' -State 'Installed' -DownloadUrl "blah2" -WebListenPrefix "blah3" -OctopusAdminCredential $null } | Should not throw
```

Comment:
Why'd this param get removed?

Reply:
Because if `$SqlDbConnectionString` is not null or empty, `$OctopusAdminCredential` cannot be null or empty.

Reply:
Ahh yeah. That makes sense.
Is it worth adding that to the test description?